src/cargo/core/compiler/job_queue.rs

```diff
@@ -1203,6 +1205,8 @@ impl<'cfg> DrainState<'cfg> {
                     config.shell().status("Documenting", &unit.pkg)?;
                 } else if unit.mode.is_doc_test() {
                     // Skip doc test.
+                } else if unit.mode.is_doc_scrape() {
+                    config.shell().status("Scraping", &unit.pkg)?;
```

Comment:
I implemented the strategy you suggested. But if you don't think it's worth it to notify the user about scraping, I can also remove the notification.

Reply:
This still doesn't seem to be working as I would expect.

In a project with `src/lib.rs` and `examples/foo.rs`, `cargo doc` doesn't mention "Scraping". I think the filter:

```rust
self.compiled.contains(&unit.pkg.package_id()) && !unit.mode.is_doc()
```

would need to be:

```rust
self.compiled.contains(&unit.pkg.package_id()) && !unit.mode.is_doc() && !unit.mode.is_doc_scrape()
```

As for whether or not to do it at all, I'm on the fence. We have felt that cargo is already a little too verbose (eventually we will likely get rid of all the status messages during a build).

If it were me, I would probably not include it.

Reply:
I just realized that it does need to print *something*, otherwise it will sometimes say "Compiling" which probably isn't good.  I think with the above adjustment, it should probably be fine.

Reply:
Ah ok I see. I've fixed it with what you suggested, and it consistently indicates when scraping.